tests/v2/validation/provisioning/rke2/provisioning_node_driver.go

```diff
@@ -72,4 +76,29 @@ func TestProvisioningRKE2Cluster(t *testing.T, client *rancher.Client, provider
 	podResults, podErrors := pods.StatusPods(client, clusterIDName)
 	assert.NotEmpty(t, podResults)
 	assert.Empty(t, podErrors)
+
+	nodesList, err := client.Management.Node.ListAll(&types.ListOpts{
+		Filters: map[string]interface{}{
+			"clusterId": clusterIDName,
+		},
+	})
+	require.NoError(&testing.T{}, err)
+	assert.NotEmpty(&testing.T{}, nodesList)
+
+	for _, rancherNode := range nodesList.Data {
+		sshkey, err := sshkeys.DownloadSSHKeys(client, rancherNode.Name)
+		require.NoError(&testing.T{}, err)
+		assert.NotEmpty(&testing.T{}, sshkey)
+		ec2Node := &nodes.Node{
+			NodeID:          rancherNode.Name,
+			PublicIPAddress: rancherNode.ExternalIPAddress,
+			SSHUser:         rancherNode.SshUser,
+			SSHKey:          sshkey,
+		}
+
+		output, err := provisioning.CheckCPU(ec2Node)
+		require.NoError(&testing.T{}, err)
```

Comment:
Tim this is wrong you need to be using the t that is passed in the function. Can you fix this in the v2.7 branch?